src/QsCompiler/SyntaxProcessor/ScopeTools.fs

```diff
@@ -52,11 +52,14 @@ type private TrackedScope =
             requiredSupport [] functors
 
 
-/// The SymbolTracker class does *not* make a copy of the given NamespaceManager,
-/// but instead will throw an InvalidOperationException upon accessing its content if that content has been modified
-/// (i.e. if the version number of the NamespaceManager has changed).
-/// The constructor throws an ArgumentException if the given NamespaceManager does not contain all resolutions,
-/// or if a callable with the given parent name does not exist in the given NamespaceManager.
+/// <summary>
+/// The <see cref="SymbolTracker"/> class does *not* make a copy of the given <see cref="NamespaceManager"/>,
```

Comment:
Adding code refs inside summary tags isn't in the original scope of this PR, but I could resist for this one :)

Reply:
Looks good. :)